.github/actions/expo-caches/action.yml

```diff
@@ -0,0 +1,173 @@
+name: 'Expo Caches'
+description: 'Composite action to perform cache related tasks.'
+
+inputs:
+  yarn-workspace:
+    description: 'Restore yarn workspace cache'
+    required: false
+  yarn-tools:
+    description: 'Restore yarn /tools cache'
+    required: false
+  yarn-docs:
+    description: 'Restore yarn /docs cache'
+    required: false
+  ios-pods:
+    description: 'Restore /ios pods cache'
+    required: false
+  bare-expo-pods:
+    description: 'Restore /apps/bare-expo/ios pods cache'
+    required: false
+  gradle:
+    description: 'Restore Gradle cache'
+    required: false
+  avd:
+    description: 'Restore AVD cache'
+    required: false
+  avd-api:
+    description: 'API version used in AVD'
+    default: '29'
+    required: false
+  ndk:
+    description: 'Restore NDK cache'
+    required: false
+  ndk-version:
+    description: 'NDK version used'
+    default: '21.4.7075529'
+    required: false
+  git-lfs:
+    description: 'Restore Git LFS cache'
+    required: false
+
+outputs:
+  yarn-workspace-hit:
+    description: 'Returns true, if yarn workspace cache is up-to-date'
+    value: steps.workspace-modules-cache.outputs.cache-hit
+  yarn-tools-hit:
+    description: 'Returns true, if yarn /tools cache is up-to-date'
+    value: steps.tools-modules-cache.outputs.cache-hit
+  yarn-docs-hit:
+    description: 'Returns true, if yarn /docs cache is up-to-date'
+    value: steps.docs-modules-cache.outputs.cache-hit
+  ios-pods-hit:
+    description: 'Returns true, if /ios pods cache is up-to-date'
+    value: steps.ios-pods-cache.outputs.cache-hit
+  bare-expo-pods-hit:
+    description: 'Returns true, if /apps/bare-expo/ios pods cache is up-to-date'
+    value: steps.bare-expo-pods-cache.outputs.cache-hit
+  avd-hit:
+    description: 'Returns true, if AVD cache is up-to-date'
+    value: steps.avd-cache.outputs.cache-hit
+  git-lfs-hit:
+    description: 'Returns true, if Git LFS cache is up-to-date'
+    value: steps.git-lfs-cache.outputs.cache-hit
+
+runs:
+  using: 'composite'
+  steps:
+    - name: 🔍️ Get yarn cache directory path
+      id: yarn-cache-dir-path
+      run: echo "::set-output name=dir::$(yarn cache dir)"
```

Comment:
We want to get rid of this cache, but I assume you just don't want to change workflows behavior right now 😉 We'll need to update these workflows though

Reply:
Yup, I don't want to modify workflows if it's not needed. There is an Android API bump and LFS cache removal on my todo list already, after this PR lands.